### PR TITLE
Git demo -001 | Action class created and POM changes.

### PR DESCRIPTION
**Pull request by SainajAnsri**

### CODE REVIEW

File: src/main/java/pack/Action_class.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package pack;
+
+public class Action_class {
+
+	public static void main(String[] args) {
+
```
Comment (SageMukhid) [inline src/main/java/pack/Action_class.java]: Add few sample code for Actions class.

File: src/main/java/pack/Action_class.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package pack;
+
+public class Action_class {
+
+	public static void main(String[] args) {
+		 
+		System.out.println(" I am java  Dev ");
+
+	}
+
```
Comment (SageMukhid) [inline src/main/java/pack/Action_class.java:12]: Formatting your code.

File: src/main/java/pack/Action_class.java (hunk 1 above)
Comment (SainajAnsri) [reply src/main/java/pack/Action_class.java]: Resolved

File: src/main/java/pack/Action_class.java (hunk 2 above)
Comment (SainajAnsri) [reply src/main/java/pack/Action_class.java:12]: Resolved

**Pull request merged**